knx/dpt/types_20.go

```diff
@@ -0,0 +1,33 @@
+package dpt
+
+// DPT_20102 represents DPT 20.102 / HVAC Mode.
+type DPT_20102 uint8
```

Comment:
Please add named constants for all values of this enum.

Like so:

```go
const (
  Auto DPT_20102 = 0
  Comfort = 1
  // ...
)
```

Reply:
Done